.travis.yml

```diff
@@ -58,12 +58,9 @@ jobs:
     - php: 7.1
       env: MOODLE_BRANCH=MOODLE_32_STABLE  DB=pgsql NODE_VERSION=8.9
 
-    - php: 7.0
-      env: MOODLE_BRANCH=MOODLE_36_STABLE  DB=pgsql
-    - php: 7.0
-      env: MOODLE_BRANCH=MOODLE_35_STABLE  DB=mysqli
-    - php: 7.0
-      env: MOODLE_BRANCH=MOODLE_34_STABLE  DB=pgsql
+    # Cannot test with PHP 7.1 any more because we are now using the :void return type-
```

Comment:
Shouldn't this comment reference to PHP 7.0 instead of PHP 7.1 ?

Reply:
Fun, because I copied it from the phpcs3 branch, that already has the change incorporated :-D 

Fixed! I've also created #93 to fix the same in the phpcs3 branch.